README.md

```diff
@@ -2048,6 +2049,214 @@ config.SetProperty(property.StatisticsPeriodSeconds.Name(), "4")
  ```
 
  After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. See the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
+ 
+### 7.8.2. Logging Configuration
+
+By default Hazelcast Go client uses DefaultLogger for logging. The default logging level is
+`info`. If you want to change the logging level for the client, you should use `LoggingLevel` property:
+
+```go
+config := hazelcast.NewConfig()
+config.SetProperty(property.LoggingLevel.Name(), logger.ErrorLevel)
+``` 
+
+As described in [Client System Properties Section](#143-client-system-properties) you can also 
+set the log level via an environment variable with this property:
+
+```go
+os.Setenv(property.LoggingLevel.Name(), logger.ErrorLevel)
+```
+
+If you are using a custom logger, `LoggingLevel` property will not be used.
+
+Possible log levels are as follows:
+```go
+// OffLevel disables logging.
+OffLevel = "off"
+// ErrorLevel level. Logs. Used for errors that should definitely be noted.
+// Commonly used for hooks to send errors to an error tracking service.
+ErrorLevel = "error"
+// WarnLevel level. Non-critical entries that deserve eyes.
+WarnLevel = "warn"
+// InfoLevel level. General operational entries about what's going on inside the
+// application.
+InfoLevel = "info"
+// DebugLevel level. Usually only enabled when debugging. Very verbose logging.
+DebugLevel = "debug"
+// TraceLevel level. Designates finer-grained informational events than the Debug.
+TraceLevel = "trace"
+```
+
+The Default Logger's format is as follows:
+
+```
+[Time] [caller method name]
+[Log level] [Client Name] [Group Name] [Client Version] [Log Message]
+```
+
+An example default log message is as follows:
+```
+2018/11/30 17:48:52 github.com/hazelcast/hazelcast-go-client/internal.(*lifecycleService).fireLifecycleEvent
+INFO:  hz.client_1 [dev] [0.4] New State :  STARTED
+```
+
+If you want to modify the Default Logger for your convenience, you can do so by accessing the embedded built-in go logger:
+```go
+l := logger.New()
+l.SetPrefix("myPrefix ")
+config := hazelcast.NewConfig()
+config.LoggerConfig().SetLogger(l)
+``` 
+
+The same log message will now be as follows:
+
+```
+myPrefix 2018/11/30 17:55:40 github.com/hazelcast/hazelcast-go-client/internal.(*lifecycleService).fireLifecycleEvent
```

Comment:
I thought that this prefix will go before the actual message "New State :  CONNECTED", don't you think so?

Reply:
This is the default behavior of built in logger. The method `SetPrefix` belongs to the built in logger. So I think this method should not be changed as users expect it to add the prefix to the beginning of each line. If we want we can add a new method that adds a prefix just before the actual message.